src/server/components/annual-review/router.ts

```diff
@@ -0,0 +1,22 @@
+import express from "express";
+import * as AnnualReviewController from "./controller";
+import { csrfRequestHandler } from "server/components/cookies/helpers";
+
+const annualReviewRouter = express.Router();
+
+annualReviewRouter.get(
+  "/annual-review/confirm/:listItemRef",
+  csrfRequestHandler,
```

Comment:
use a wildcard route `annualReviewRouter.all('/annual-review*', csrfRequestHandler)` to apply csrf to all the routes

Reply:
We might need to make this eslint rule a warning 🤷🏿‍♂️ 
![Uploading Screenshot 2022-12-14 at 15.22.15.png…]()

Reply:
I guess because it's using `listRouter.get("/", listsController);` as a route and because no `listId` is being passed in as a param it doesn't produce the data in locals

Reply:
oh yeah, go ahead if you haven't already. I think in other files it's using an ignore comment